Add section about accessing afs folder of an other user

mevouc opened the pull request.

Commit:
Add section about accessing afs folder of an other user

Pluggi commented:
If you could add a clarification about the new logins it could be cool.
Indeed, if the login contains a dot, like "john.doe", the user has to execute "kinit john_doe".

cri-epita commented:
NO ! This is not "kinit john-doe" but "kinit john_doe".

mevouc committed:
Add clarification for new logins

I chose to simply add one sentence for this clarification. Tell me if you want me to rewrite the full process instead.

mevouc commented:
Updated. Thank for the feedback.
